build/zip.py

```diff
@@ -57,7 +61,8 @@ def main(argv):
   with open(runtime_deps) as f:
     for dep in f.readlines():
       dep = dep.strip()
-      dist_files += [dep]
+      if dep not in dist_files:
+        dist_files += [dep]
```

Comment:
better: just make `dist_files` a set.

Reply:
I wasn't sure if `dist_files` needed to maintain a certain order that set's don't guarantee

cc @jkleinsc ❓

Reply:
@MarshallOfSound I don't think order matters

Reply:
yes, order doesn't matter.